charts/agent/templates/daemonset.yaml

```diff
@@ -1,4 +1,7 @@
 {{- if .Values.daemonset.deploy }}
+{{- if and .Values.gke.autopilot .Values.slim.enabled }}
```

Comment:
use `{{- include "agent.gke.autopilot" . }}` to check for autopilot using the helper.